aep/general/0003/aep.md.j2

```diff
@@ -99,6 +105,22 @@ Examples of complexities that declarative clients abstract away include:
 
 [Terraform][] is an example of such a client.
 
+### Root collection
+
+A [collection](#collection) that does not have a parent collection. For example,
+`/users/`.
+
+### Root Resource
+
+A `root` resource is a resource which has a collection that has no parents. In
+other words, it is a resource that has a collection that appears at the root of
+a resource hierarchy.
+
+A root resource may also appear as a child of another resource: for example, a
+user permission may exist globally, but also as a child of a individual object.
+In this case, the resource is still considered a "root" resource, because at
```

Comment:
We have both `root` (backticked) and "root" (quoted) in this passage.  Seems like we should be consistent about this.  Maybe we just don't use any punctuation?

Reply:
agreed. I think we can just do away with any punctuation, will fix.